i/cont_filter_cache/cont_filter_cache_ubi_8.5.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash -e
+
+# ----------------------------------------------------------------------------
+#
+# Package        : github.com/IBM/controller-filtered-cache
```

Comment:
I think we just give the repository name as the Package name for most scenarios?